x-pack/plugins/gis/public/shared/layers/sources/ems_tms_source.js

```diff
@@ -105,6 +105,22 @@ export class EMSTMSSource extends TMSSource {
     return this._descriptor.id;
   }
 
+  async getAttributions() {
+    const service = this._getTMSOptions();
+    const attributions = service.attributionMarkdown.split('|');
+
+    return attributions.map((attribution) => {
+      attribution = attribution.trim();
+      //this assumes attribution is plain markdown link
```

Comment:
fallback for current version of EMS which still uses markdown to configure attribution